build_docker.sh

```diff
@@ -32,6 +32,7 @@ PUSH="${PUSH:-false}"
 TARGET="${TARGET:-${DEFAULT_TARGET}}"
 TAGS="${TAGS:-${DEFAULT_TAGS}}"
 BASE="${BASE:-${DEFAULT_BASE}}"
+PLATFORM="${PLATFORM:-}" # default to all platforms
```

Comment:
Our build infra calls this script (not the make targets) and defaulting to "" means that we keep the existing behaviour where mkctr builds for all platforms it supports https://github.com/tailscale/mkctr?tab=readme-ov-file#usage